mri/reconstructors/self_calibrating.py

```diff
@@ -217,7 +218,9 @@ def reconstruct(self, kspace_data, optimization_alg='pogm', x_init=None,
                 n_cpu=self.n_jobs
             )
             self.set_smaps(Smaps)
+            print('smaps extracted')
         # Start Reconstruction
+        print('start reconstruction')
```

Comment:
And here. I know that this could sometimes be slow step, but any prints that we have needs to be shielded.